src/style.css

```diff
@@ -0,0 +1,103 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 11px;
+  line-height: 13px;
+  color: #000;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  padding: 30px 14px 14px;
+
+  border: 1px solid #f3f3f3;
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.07);
+  border-radius: 5px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 5px 45px;
+}
+
+.card__title {
+  margin-bottom: 5px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.card__subtitle {
+  margin-bottom: 15px;
+
+  font-weight: 400;
+  font-size: 11px;
+}
+
+.reviews {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+.reviews__stars {
+  display: flex;
+}
+
+.stars__item {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+
+  background: url(images/star.svg) no-repeat center;
+}
+
+.stars__item:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+.price__numbers {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+.card__button {
+  height: 40px;
+  width: 170px;
+  padding-top: 12px;
+  
+  font-weight: 700;
+  font-size: 14px;
+  text-transform: uppercase;
+  text-align: center;
+  color: #fff;
+  text-decoration: none;
+
+  border-radius: 5px;
+  background: #00acdc;
+}
+
+.card__button:hover {
+  border: 1px solid #00acdc;
```

Comment:
don't add border on hover. Add it to default button state and make it transparent, if you need such behaviour. On hover just change it's color